### PR TITLE
Remove redundant code, makes changes to webAPI documentation

### PR DESCRIPTION
**Pull request by abhiabhi94**
Make changes left behind in 522db6f

### CODE REVIEW

File: docs/source/Web API.rst [hunk 1]
```diff
@@ -87,12 +87,12 @@ Get request accepts 3 params:
...
+For example if you are using axios to retrieve the comment list of second object (id=2) of a model (content type) called post inside the app(django app) post.
 you can do the following:
 
 ::
 
-    $ curl -H "Content-Type: application/json" 'http://localhost:8000/api/comments/?type=MODEL_NAME&id=ID'
+    $ curl -H "Content-Type: application/json" 'http://localhost:8000/api/comments/?model_name=MODEL_NAME&model_id=ID&app_name=APP_NAME'
 
 
 **2- Create a comment or reply to an existing comment:**
```
Comment (Radi85) [inline docs/source/Web API.rst:98]: This section needs to be updated.

File: README.rst [hunk 2]
```diff
@@ -447,20 +447,21 @@ Get request accepts 3 params:
...
 **2- Create a comment or reply to an existing comment:**
 
 Authorization must be provided as a TOKEN or USERNAME:PASSWORD.
 
-- ``type``: is the model name of the content type that have comments associated with it.
-- ``id``: is the id of an object of that model
+- ``model_name``: is the model name of the content type that have comments associated with it.
+- ``app_name``: the name of the app that contains that model.
+- ``model_id``: is the id of an object of that model
 - ``parent_id``: is 0 or **NOT PROVIDED** for parent comments and for reply comments must be the id of parent comment
```
Comment (Radi85) [inline README.rst:467]: The section below needs to be updated.

**Review by Radi85 — CHANGES_REQUESTED** [review]
If you would, you can revert the changes in the documentation and I will take care of it.

File: README.rst (hunk 2 above)
Comment (abhiabhi94) [reply README.rst:467]: is this not correct in its current state?

Comment (Radi85) [reply README.rst:467]: It's correct but there might be few lines need to be updated for now I can see this example:
```bash
$ curl -X POST -u USERNAME:PASSWORD -d "content=CONTENT" -H "Content-Type: application/json" "http://localhost:8000/api/comments/create/?type=MODEL_NAME&id=ID&parent_id=0"
```

Comment (abhiabhi94) [reply README.rst:467]: yes, you're right. i have updated wherever i found the changes need to be made.

**Review by Radi85 — APPROVED** [review]

**Pull request merged**